### PR TITLE
2.3_3.1_3.2_3.3_3.4_4.1_4.2_5.1  add modify delete new product

### PR DESCRIPTION
**Pull request by valeriia-danylenko**
Admin MVC

Added Routes:
adminRouter.get("/all-salesmen")
adminRouter.get("/products");
adminRouter.get("/products/add")
adminRouter.post("/products/add");
adminRouter.get("/products/:id")
adminRouter.put("/products/:id");
adminRouter.delete("/products/:id")
productsRouter.get('/all-categories')
productsRouter.get('/all-manufactures')
productsRouter.get('/all-units')

Added functionality (
- get all categories/manufactures/units/vendors for a dropdown menu
- add new products and assing its salesman
- update product information by id
- delete product by id )

### CODE REVIEW

**Review by DianaBondarenko — COMMENTED** [review]
Good job!!!

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```
Comment (AllaYakymova) [inline .vscode/settings.json]: remove this file

File: Dp-202-shop/server/src/db/models/Order.model.ts [hunk 2]
```diff
@@ -8,8 +8,6 @@ export interface OrderAttributes {
   total_sum: number;
   status?: string;
   id?: number;
-  buyer_id?: number;
-  salesman_id?: number;
   createdAt?: Date;
   updatedAt?: Date;
```
Comment (AllaYakymova) [inline Dp-202-shop/server/src/db/models/Order.model.ts]: don't delete anything from db without informing. Put it back. It's the fields that are not necessary to enter

File: Dp-202-shop/server/src/db/models/OrderItem.model.ts [hunk 3]
```diff
@@ -7,17 +7,15 @@ import {Order} from "./Order.model";
 export interface OrderItemAttributes {
   quantity: number;
   id?: number;
-  order_id?: number;
-  product_id?: number;
   createdAt?: Date;
   updatedAt?: Date;
```
Comment (AllaYakymova) [inline Dp-202-shop/server/src/db/models/OrderItem.model.ts]: don't delete anything from db without informing. Put it back. It's the fields that are not necessary to enter

File: Dp-202-shop/server/src/db/models/OrderItem.model.ts [hunk 4]
```diff
@@ -7,17 +7,15 @@ import {Order} from "./Order.model";
...
 }
 
 interface OrderItemCreationAttributes extends Optional<OrderItemAttributes, 'id'> {}
 
-@Table({ tableName: 'order_items' })
+@Table({ tableName: 'order-items' })
 export class OrderItem extends Model<OrderItemAttributes, OrderItemCreationAttributes> {
-  public static readonly tableName: string = 'order_items';
+  public static readonly tableName: string = 'order-items';
```
Comment (AllaYakymova) [inline Dp-202-shop/server/src/db/models/OrderItem.model.ts]: don't delete anything from db without informing. Put it back.

File: Dp-202-shop/server/src/db/models/Product.model.ts [hunk 5]
```diff
@@ -9,6 +9,10 @@ import {OrderItem} from "./OrderItem.model";
 
 export interface ProductAttributes {
   vendor_code: number;
+  // category_id? : number;
+  // manufacture_id?: number;
+  // user_id?: number;
+  // unit_id?: number;
   product_name: string;
```
Comment (AllaYakymova) [inline Dp-202-shop/server/src/db/models/Product.model.ts]: don't delete anything from db without informing. Put it back. It's the fields that are not necessary to enter

File: Dp-202-shop/server/src/db/models/Order.model.ts [hunk 6]
```diff
@@ -8,8 +8,6 @@ export interface OrderAttributes {
   total_sum: number;
   status?: string;
   id?: number;
-  buyer_id?: number;
```
Comment (DianaBondarenko) [inline Dp-202-shop/server/src/db/models/Order.model.ts]: Why buyer_id and salesman_id are deleted?

**Review by DianaBondarenko — APPROVED** [review]

**Pull request merged**